### DISCUSSION TITLE
Text scrolling or wrapping in text based skin elements

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#28866] [Ideas]
Hello, 
Sometimes, dynamic text in skins becomes way to long (like song names for example), it would be nice to have the option to select a wrapping style in the skin editor. Like making the text scroll, or wrap to the next line.

Thanks for reading !

**Comment by shiumano** [comment]
Personally, I would also like to have text color and background color settings.
I am not sure if I should separate this into a separate thread or if this thread should be expanded.
